smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -205,13 +205,15 @@ private String getThrownExceptions() {
     }
 
     private void generateCommandConstructor() {
-        writer.openBlock("constructor(readonly input: $T) {", "}", inputType, () -> {
-            // The constructor can be intercepted and changed.
-            writer.write("// Start section: $L", COMMAND_CONSTRUCTOR_SECTION)
-                    .pushState(COMMAND_CONSTRUCTOR_SECTION)
-                    .write("super();")
-                    .popState()
-                    .write("// End section: $L", COMMAND_CONSTRUCTOR_SECTION);
+        writer
+            .writeDocs("@public")
```

Comment:
nit: for chained method calls, the first method call shouldn't be on the next line.

Reply:
Fixed in https://github.com/awslabs/smithy-typescript/pull/719/commits/97b97326df590ad73d01a0de396421328a77ffb3